packages/react-ui-ag/src/Gmap/Marker.js

```diff
@@ -16,7 +15,6 @@ export default class Marker extends PureComponent {
 
   get marker() {
     return props => ({
-      icon: markerRedDotIcon(),
```

Comment:
Why did you remove this?

Reply:
He just renamed it. I like the new name. Less Smurfy.

Reply:
It was removed before from what I saw, I dunno 🤷‍♂️